src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/info-card/info-card.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="col-md-6">
```

Comment:
Having `<div class="col-md-6">` inside the component template is very limiting. I recommend that this shoudl be declared outsite, by the one that uses the component:

```
<div class="col-md-6">
  <cd-info-card ...
...
```

Reply:
I agree. I'll implement it.